### ISSUE TITLE
Error at installation time

### ISSUE THREAD

**Issue by f0nzie** [workflowr/workflowr#88]
When installing with the `build_vignettes` option TRUE, there is an error. This occurs regardless if it is a project with a git repo or a new project without one.

```
* preparing 'workflowr':
* checking DESCRIPTION meta-information ... OK
* installing the package to build vignettes
* creating vignettes ...Warning: running command '"C:/PROGRA~1/R/R-34~1.3/bin/x64/Rscript" --vanilla --default-packages= -e "tools::buildVignettes(dir = '.', tangle = TRUE)"' had status 1
 ERROR
Quitting from lines 94-97 (wflow-01-getting-started.Rmd) 
Error: processing vignette 'wflow-01-getting-started.Rmd' failed with diagnostics:
The directory where you have chosen to create a new workflowr directory is already within a Git repository. This is potentially dangerous. If you want to have a workflowr project created within this existing Git repository, re-run wflow_start with `git = FALSE` and then manually commit the new files. The following directory contains the existing .git directory: C:/Users/msfz751
Execution halted
```

**Comment by jdblischak** [comment]
Thanks for reporting this @f0nzie, and I apologize for the inconvenience. I recently learned about this issue on Windows (see #87). The short-term [solution](https://github.com/jdblischak/workflowr/issues/87#issuecomment-350292274) is to run `devtools::install_github("jdblischak/workflowr")`.

I think the bug is due to how Windows handles the path to `/tmp`. I have a fix planned, and I'll let you know once this is resolved.

**Comment by jdblischak** [comment]
@f0nzie Is your home directory a Git repository? I suspect this is causing the issue. Could you please run the following in the R console and post the output?

```
dir.exists("C:/Users/msfz751/.git")
file.minfo("C:/Users/msfz751/.git")
```

I don't think workflowr would have erroneously created a Git repo in your home directory, but it's possible (which is why I am having you check the modification time). I am still investigating. In the meantime, you should delete the Git repo in your home directory since you most likely do not want this:

```
unlink("C:/Users/msfz751/.git", recursive = TRUE, force = TRUE)
```

**Comment by jdblischak** [comment]
@f0nzie This problem will be avoided in the just released version 0.9.0 (see https://github.com/jdblischak/workflowr/commit/f91faa81d79e5961c755b446c56d2b0c46db00ae). The code in the vignette will only be evaluated if it is possible to do so without errors.

If you have the time to debug, I'd still be interested in the answers to my previous questions, so please feel free to continue posting here even though the Issue is closed.
